.github/workflows/pull-request-title.yml

```
@@ -9,5 +9,5 @@ jobs:
     steps:
     - uses: fastify/action-pr-title@v0
       with:
-        prefixes: 'build:,chore:,ci:,docs:,feat:,fix:,perf:,refactor:,style:,test:'
-        github-token: ${{ github.token }}
\ No newline at end of file
+        regex: '/^(build|chore|ci|docs|feature|types|fix|perf|refactor|style|test){1}([ ]{0,1}\([^\):]*\)){0,1}:/'
```

```suggestion
        regex: '/^(build|chore|ci|docs|feature|types|fix|perf|refactor|style|test)(?:\([^\):]*\))?:\s/'
```

- The {1} after the commit message prefix group is redundant. The default behavior is to match once.
- Replace ([ ]{0,1}) with \s to match any whitespace character (spaces, tabs, etc.) after the prefix to make it more flexible.
- Modify the optional capturing group (?:\([^\):]*\))? to handle any parentheses with content within it (e.g., (some-text-here)) that might come after the prefix.
- Add \s after the optional parentheses capturing group to match any additional whitespace between the colon and the actual commit message.

Matching titles:

- "docs: Update README.md"
- "fix: Fix a bug in the login process"
- "test(some-context): Add unit tests for the API"
- "chore(build): Update build script"